buildSrc/src/main/java/com/google/firebase/gradle/plugins/ModuleVersion.kt

```diff
@@ -57,17 +57,17 @@ enum class PreReleaseVersionType {
  *
  * Note that `build` will always be present as starting at one by defalt. That is, the following
  * transform occurs:
- *
  * ```
  * "12.13.1-beta" // 12.13.1-beta01
  * ```
  *
- * @see fromStringsOrNull
- *
  * @property type an enum of [PreReleaseVersionType] that identifies the pre-release identifier
  * @property build an [Int] that specifies the build number; defaults to one
+ * @see fromStringsOrNull
```

Comment:
This breaks the popup rendering (when you hover over the class), but ironically doesn't break the inline rendering. The fix is having it above the `@property` annotations :')

Reply:
Ah, unsure what moved these.